### PR TITLE
New Template: alcatel_sros, sh router mpls lsp

### PR DESCRIPTION
**Pull request by h4ndzdatm0ld**
- New Template Pull Request

alcatel_sros_show_router_mpls_lsp,
alcatel_sros,
"show router mpls lsp"

### CODE REVIEW

File: ntc_templates/templates/alcatel_sros_show_router_mpls_lsp.textfsm [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Value Required LSP_NAME (\S+)
+Value Required TO (\d+.\d+.\d+.\d+)
+Value Required TUN_ID (\d+)
+Value Required FASTFAIL_CONFIG (\S+)
+Value Required ADM (Up|Dwn)
+Value Required OPR (Up|Dwn)
+
+Start
```
Comment (FragmentedPacket) [inline ntc_templates/templates/alcatel_sros_show_router_mpls_lsp.textfsm:8]:
```suggestion
Start
  ^=+
```

File: ntc_templates/templates/alcatel_sros_show_router_mpls_lsp.textfsm [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  ^=.
+  ^MPLS\s+LSPs\s+\S+
```
Comment (FragmentedPacket) [inline ntc_templates/templates/alcatel_sros_show_router_mpls_lsp.textfsm:10]:
```suggestion
  ^MPLS\s+LSPs
```
You don't need to account for the whole line if it's unique enough

File: ntc_templates/templates/alcatel_sros_show_router_mpls_lsp.textfsm [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  ^LSP\s+Name\s+Tun\s+Fastfail\s+Adm\s+Opr\s*$$
```
Comment (FragmentedPacket) [inline ntc_templates/templates/alcatel_sros_show_router_mpls_lsp.textfsm:11]:
```suggestion
  ^LSP\s+Name\s+Tun\s+Fastfail\s+Adm\s+Opr\s*$$ -> LSP
```

Comment (FragmentedPacket) [reply ntc_templates/templates/alcatel_sros_show_router_mpls_lsp.textfsm:11]: Move the other lines into the **LSP** state as well.

**Review by FragmentedPacket — CHANGES_REQUESTED** [review]

**Comment by h4ndzdatm0ld** [comment]
Thanks @FragmentedPacket + Updates / Passing tests

**Review by h4ndzdatm0ld — COMMENTED** [review]
Looks good

**Review by FragmentedPacket — APPROVED** [review]

File: ntc_templates/templates/index [hunk 4]
```diff
@@ -21,6 +21,11 @@ alcatel_sros_show_router_mpls_interface.textfsm, .*, alcatel_sros, sh[[ow]] rout
 alcatel_sros_show_router_rsvp_interface.textfsm, .*, alcatel_sros, sh[[ow]] router rsvp int[[erface]]
 alcatel_sros_show_router_ldp_interface.textfsm, .*, alcatel_sros, sh[[ow]] router ldp int[[erface]]
 alcatel_sros_show_router_pim_interface.textfsm, .*, alcatel_sros, sh[[ow]] router pim int[[erface]]
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (h4ndzdatm0ld) [reply ntc_templates/templates/index]: Okie dokie ^_^

**Comment by FragmentedPacket** [comment]
This has some changes from `master` included in it now. Let me know if you want me to help with getting it resolved.

**Pull request merged**